mock/py/mockbuild/util.py

```diff
@@ -714,6 +714,7 @@ def do_with_status(command, shell=False, chrootPath=None, cwd=None, timeout=0, r
             command = _prepare_nspawn_command(chrootPath, user, command,
                                               nspawn_args=nspawn_args,
                                               env=env, cwd=cwd, shell=shell)
+            env['SYSTEMD_NSPAWN_TMPFS_TMP'] = '0'
```

Comment:
can we rather pass it to _prepare_nspawn_command. Or even better - set this inside of _prepare_nspawn_command().

Reply:
I can rewrite this to return tuple, `command` + `env_overrides`.  I'm not sure
it is wise to edit `env` that is passed by reference ... wdyt?

Reply:
What about setting this inside of _prepare_nspawn_command?

Reply:
That would mean passing `env` down into _prepare_nspawn_command
by reference, and editing it there.  Perhaps yes, ... I wanted to do this but
then I realized that we have to flip `shell` from `True` to `False` after calling
this method anyways - and we can not pass immutable down by reference
(bool).  So to make it nice, it only makes sense to fix both env and shell 
variables.